src/main/kotlin/no/nb/mlt/wls/infrastructure/callbacks/NotificationItemPayload.kt

```diff
@@ -9,66 +9,83 @@ import no.nb.mlt.wls.domain.model.ItemCategory
 import no.nb.mlt.wls.domain.model.Owner
 import no.nb.mlt.wls.domain.model.Packaging
 
+@Schema(
+    description = """Payload for updates about items sent from Hermes WLS to the appropriate catalogues.""",
+    example = """
+    {
+      "hostId": "mlt-12345",
+      "hostName": "AXIELL",
+      "description": "Tyven, tyven skal du hete",
+      "itemCategory": "PAPER",
+      "preferredEnvironment": "NONE",
+      "packaging": "NONE",
+      "owner": "NB",
+      "callbackUrl": "https://callback-wls.no/item",
+      "location": "SYNQ_WAREHOUSE",
+      "quantity": 1
+    }
+    """
+)
 data class NotificationItemPayload(
     @Schema(
-        description = "ID from the host system which owns the item.",
-        examples = ["item-12345"]
+        description = """The item ID from the host system, usually a barcode or an equivalent ID.""",
+        example = "item-12345"
     )
     val hostId: String,
     @Schema(
-        description = "Name of the host system which manages the item.",
+        description = """Name of the host system which the item originates from.
+                Host system is usually the catalogue that the item is registered in.""",
         examples = ["AXIELL", "ALMA", "ASTA", "BIBLIOFIL"]
     )
     val hostName: HostName,
     @Schema(
-        description =
-            "Description of the item for easy identification in the warehouse system. " +
-                "Usually an item title/name, e.g. book title, film name, etc. or contents description.",
+        description = """Description of the item for easy identification in the warehouse system.
+                Usually an item title/name, e.g. book title, film name, etc. or contents description.""",
         examples = ["Tyven, tyven skal du hete", "Avisa Hemnes", "Kill Buljo"]
     )
     val description: String,
-    // TODO - Update this schema to reflect new categories
     @Schema(
-        description = "What kind of item category the item belongs to, e.g. Books, Issues, Films, etc.",
-        examples = [
-            "safetyfilm", "nitratfilm", "film", "plater", "fotografier", "papir",
-            "gjenstand", "lydbånd", "videobånd", "sekkepost", "magnetbånd"
-        ]
+        description = """Item's category, same category indicates that the items can be stored together without any preservation issues.
+                For example: books, magazines, newspapers, etc. can be stored together without damaging each other.""",
```

Comment:
Same comment as before regarding specifying category - missed this in the first review round